### DISCUSSION TITLE
prisma 暂不支持 postgrep 的 ltree 类型，希望可以添加这个类型，可以快速进行树查询操作

### DISCUSSION THREAD

**Discussion by shanwenjie** [prisma/prisma#9741] [Q&A]
prisma 暂不支持 postgrep 的 ltree 类型，希望可以添加这个类型，可以快速进行树查询操作 ？？？？

**Comment by janpio** [comment]
There is an existing feature request for ltree support in Prisma: https://github.com/prisma/prisma/issues/2568

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
